tests/rados/test_osd_heartbeat.py

```diff
@@ -26,7 +26,7 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosOrchestrator(node=cephadm)
     timeout = config.get("timeout", 150)
-    recovery_timeout = config.get("recovery_timeout", 500)
+    recovery_timeout = config.get("recovery_timeout", 2400)
```

Comment:
40 mins is too big of a timeout to let a machine restart and come back online, 10/15 mins should be more than enough IMO

Reply:
I believe it's safe to add this, as the wait is not a blind sleep, and the status is checked periodically until the timeout. So I think it's fine.